v2/internal/providers/kubernetes.go

```diff
@@ -2,7 +2,7 @@ package providers
 
 import (
 	"context"
-	"github.com/datadog/stratus-red-team/internal/utils"
+	"github.com/datadog/stratus-red-team/v2/internal/utils"
```

Comment:
Just to consistent with other use, shouldn't we call this `utils2`?

```suggestion
         utils2 "github.com/datadog/stratus-red-team/v2/internal/utils"
```

Reply:
There is a use case on lines https://github.com/DataDog/stratus-red-team/pull/146/files#diff-8106b62550d342b69bc5acbffc4431b731809ded0e93090f12ae7b99092b79b1L58-R60

Reply:
fixed in [b38133c](https://github.com/DataDog/stratus-red-team/pull/146/commits/b38133c9bbeba4ae777a49daec5c79d957abf752)